### ISSUE TITLE
Source Search not only just "All and Pinned"

### ISSUE THREAD

**Issue by NeKoOuO** [mihonapp/mihon#92]
### Describe your suggested feature

Assuming that I have too many sources, causing each search to query a large number of sources, resulting in significant lag.
I hope to be able to group these sources and only select one group for searching.
Not just Pinned and All
https://github.com/jobobby04/TachiyomiSY/issues/959

### Other details

The source search function should not only be divided into All and Pinned
I hope to integrate categories and the search function and be able to choose which group to search.

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open or closed issue.
- [X] I have written a short but informative title.
- [X] If this is an issue with an official extension, I should be opening an issue in the [extensions repository](https://github.com/tachiyomiorg/extensions/issues/new/choose).
- [X] I have updated the app to version **[0.16.0](https://github.com/mihonapp/mihon/releases/latest)**.
- [X] I will fill out all of the requested information in this form.

**Comment by TheColorman** [comment]
Use case would probably be that you have 15 extensions for manga, 15 for manwha. Checking each one would take too long, and searching all of them would be wasteful.

**Comment by NeKoOuO** [comment]
@MajorTanya I categorize based on diverse source languages and genres of manga/manhua/manhwa.

**Comment by MajorTanya** [comment]
> @MajorTanya I categorize based on diverse source languages and genres of manga/manhua/manhwa.

I understand now, my bad for earlier. Although I think this feature would require a significant amount of reworking in the background to make happen, so I wouldn't hold my breath for this to be realised

**Comment by MikeZeDev** [comment]
Would it be possible to get a setting that just default the search to "All". I dont have a preference for any website, i just have a couple of them, nothing is pinned so defaulting to pinned is kinda annoying. 

Dont wanna put my eggs in the same basket.

**Comment by NeKoOuO** [comment]
> Would it be possible to get a setting that just default the search to "All". I dont have a preference for any website, i just have a couple of them, nothing is pinned so defaulting to pinned is kinda annoying. 
> 
> Dont wanna put my eggs in the same basket.
> 
> 
For me, I have as many as more than thirty sources. Every time I search, if it is a search to cover all, the waiting time will be very long. I will classify it according to the language, but currently, I can only search for the pinned sources, unable to choose my own classified group.

**Comment by NeKoOuO** [comment]
> Calm down, i've talked about an **option**. No need to be upset for something that wont affect you. That's a misunderstanding.
> It should not be a big deal to just allow changing the default stuff.
> 
> You know how it is. If you let pinned as default, and just make an option most people will stay on pinned, that how options works so it could be ok
> 
Why not pinned all your sources if you don't have many sources??

**Comment by AntsyLich** [comment]
> You know how it is. If you let pinned as default, and just make an option most people will stay on pinned, that how options works so it could be ok

I can already imagine people making videos on YouTube "Fix missing source while searching in Mihon. 2024 updated. 109% working"

**Comment by MikeZeDev** [comment]
I think you underestimate the lazyness of people , and the fact they most dont change default options but if you dont want to do itthat its fine. No need to be angry about it 👌

**Comment by chewwie-hime** [comment]
I agree with MikeZeDev, the Pinned sources thing is so annoying. I don't have a specific source everytime I look for things and prefer to have everything listed. At least have it as an option to change the default search settings.

**Comment by AntsyLich** [comment]
> I don't have a specific source everytime I look for things 

Neither do I. I have well over 20 sources and I have pinned 5. I get all my stuff on that 5 sources. When I don't i search all the sources.

**Comment by DarkOps3** [comment]
I only care about the source categories and not the search part, should I make a new issue?

**Comment by MajorTanya** [comment]
> I only care about the source categories and not the search part, should I make a new issue?

No, as you would know if you searched existing issues and found #1383 (also linked right above your comment). Source-based categories as a feature request will stay in here for the time being,
